.github/workflows/CI.yml

```diff
@@ -138,6 +138,12 @@ jobs:
           command: fmt
           args: --all -- --check
 
+  cargo-deny:
+    runs-on: ubuntu-latest
```

Comment:
Is it possible to in addition run this every day or once a week or so?

Reply:
It should be yes, I will look into it. But not sure if we will receive some form of notification when it fails

Reply:
I get a mail whenever a build on `master` fails, so I would expect that to happen. To be checked :)

Reply:
Should we just merge this for now and worry about the scheduled job independently? I don't know how hard that is to add.

Reply:
It should be easy, I just didn't had the time for it yet. Hopefully this weekend

Reply:
Let's merge it as-is for now then? @bilelmoussaoui

Reply:
yeah, sounds good to me